README.md

```diff
@@ -30,45 +35,77 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Data Science](https://medium.com/ibm-watson): Systems and scientific methods to analyze structured and unstructured data in order to extract knowledge and insights.
 * [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
 
-# Watch the Video
-
 # Steps
 
-## Run locally
+Either Run locally:
 
 1. [Clone the repo](#1-clone-the-repo)
-1. [Run the notebook](#2-run-the-notebook)
+2. [Run Jupyter Notebooks](#2-run-jupyter-notebooks)
+
+or in Watson Studio:
+
+3. [Sign up for the Watson Studio](#3-sign-up-for-watson-studio)
+4. [Create the notebook](#4-create-the-notebook)
+
+then:
+
+5. [Run the Notebook](#5-run-the-notebook)
+
+## Run locally
 
 ### 1. Clone the repo
 
-Clone the `xxxxxxxx` locally. In a terminal, run:
+Clone the `https://github.com/IBM/ensure-loan-fairness-aif360` locally. In a terminal, run:
```

Comment:
I'd remove the back ticks so the link gets link formatting, but it's ok if you like this way